german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/gruessen_freundlich.md

```diff
@@ -5,3 +5,7 @@ ihr Gesicht geschrieben. Gerade als Sie ...
 "Du darfst nicht hier sein!", zischt Ihnen die junge Frau zu. Feuchte, dunkle Haarsträhnen hängen
 ihr ins Gesicht, darunter blitzen merkwürdig schwarze Augen im Feuerschein. Etwas scheint außerdem
 mit dem Mann nicht zu stimmen. Nach einem Augenblick wird Ihnen klar, dass er blind ist.
+
+Sie entschließen sich 
+
+[dazu vor den beiden abzuhauen](https://github.com/ll9/create-your-own-adventure/blob/some-changes/german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/abhauen/abhauen.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.